### PR TITLE
Implement OSSCheck on CI

### PR DESCRIPTION
**Pull request by jpsim**
Work in progress to address #1025.

### CODE REVIEW

**Comment by jpsim** [comment]
@marcelofabri what do you think of this as a first version of OSSCheck?

**Comment by marcelofabri** [comment]
Are you Santa? 🎅🎁 

This is an awesome first step. I'd only prefer that we did the building and running on another script and then Danger would only be used to report the differences. Does that make sense? This way we could easily run OSSCheck locally as well.

**Comment by marcelofabri** [comment]
Also using the JSON report might be better than the Xcode one.

**Comment by jpsim** [comment]
> I'd only prefer that we did the building and running on another script and then Danger would only be used to report the differences. Does that make sense? This way we could easily run OSSCheck locally as well.

That's how I had first prototyped this, but the fact that this mangles the local git environment (by checking out `master`, pulling, and updating submodules) meant that I wanted to keep that to a CI-only script for now.

> Also using the JSON reporter might be better than the Xcode one.

Not with the way this is currently built, since it relies on each violation being in a single line, doing a set difference on the array of lines to report fixed/introduced violations.

> Are you Santa? 🎅🎁

Yes. 💃

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,13 @@
 language: generic
 matrix:
   include:
-    - script:
```
Comment (marcelofabri) [inline .travis.yml]: commenting just as a reminder that this should be added again before merging

**Comment by jpsim** [comment]
I'd say this is a fine starting point. Let's refine and improve over time. 😄

**Comment by marcelofabri** [comment]
🎉

**Pull request merged**